R/data.R

```diff
@@ -83,4 +83,39 @@ NULL
 #'
 NULL
 
-
+#' @title SilvermanAGutData
+#' @description The SilvermanAGutData dataset contains 16S rRNA gene based high-throughput 
```

Comment:
Same here, we can omit @description tag. The text that comes after the first row (title) is interpreted as description.

Reply:
Ah. Ok let's use the same everywhere. Easier to do same as the already existing. If we change later we can change all at once.